### PR TITLE
Partial tile suffixing added to the path libs

### PR DESCRIPTION
**Pull request by mhutchinson**

### CODE REVIEW

File: api/layout/paths.go [hunk 1]
```diff
@@ -29,20 +29,32 @@ const (
...
+	p := partialTileSize(0, seq, logSize)
+	return EntriesPath(seq/256, p)
 }
 
-// EntriesPath returns the local path for the Nth entry bundle.
-func EntriesPath(N uint64) string {
-	return fmt.Sprintf("tile/entries%s", fmtN(N))
+// EntriesPath returns the local path for the nth entry bundle. p denotes the partial
+// tile size, or 0 if the tile is complete.
+func EntriesPath(n, p uint64) string {
```
Comment (AlCutter) [inline api/layout/paths.go:41]: Should we make this take `logSize` too for consistency?

Comment (mhutchinson) [reply api/layout/paths.go:41]: I had some reason for doing it this way, but I think that reason is bogus. I'll take a look at this.

Comment (mhutchinson) [reply api/layout/paths.go]: Done.

**Comment by mhutchinson** [comment]
@AlCutter ready for review again

**Review by AlCutter — APPROVED** [review]

**Pull request merged**